generator/integration_tests/golden/internal/golden_kitchen_sink_stub.h

```diff
@@ -71,6 +72,12 @@ class GoldenKitchenSinkStub {
       google::cloud::CompletionQueue const& cq,
       std::unique_ptr<grpc::ClientContext> context) = 0;
 
+ virtual std::unique_ptr<::google::cloud::internal::StreamingWriteRpc<
+      google::test::admin::database::v1::WriteObjectRequest,
+      google::test::admin::database::v1::WriteObjectResponse>>
+  WriteObject(
+      std::unique_ptr<grpc::ClientContext> context) = 0;
+
```

Comment:
Aside: It looks like we're generating a post-declaration separator here, when we don't know if we actually need one.

Reply:
Yes. Some of the generators use pre-declaration/definition separators, others use post.  I went with what was already there (at least I hope I did).  I agree that pre-declaration/definition separators are easier to reason about.